ext/strings.go

```diff
@@ -1000,8 +1005,9 @@ func formatHex(useUpper bool) clauseImpl {
 		case types.StringType, types.BytesType:
 			if arg.Type() == types.BytesType {
 				return fmt.Sprintf(fmtStr, arg.Value().([]byte)), nil
+			} else {
```

Comment:
nit: please revert to the de-dented return below

Reply:
Gotcha, I've de-dented this block.